frontend-react/src/components/Admin/EditSenderSettings.test.tsx

```diff
@@ -86,10 +86,10 @@ describe("EditSenderSettings", () => {
     });
 
     test("should be able to edit processing type field", async () => {
-        const container = render(
+        const view = render(
             <EditSenderSettings {...mockRouteComponentProps} />
```

Comment:
Seems like the linter still isn't a fan of this method; wants us to use `screen` imported from the testing utility library.

Reply:
fixed